sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -153,12 +153,12 @@ describe("SmsClient [Playback/Live]", async () => {
       );
       assert.fail("Should have thrown an error");
     } catch (e) {
-      assert.equal(e.statusCode, 400);
+      assert.equal(e.statusCode, 404);
```

Comment:
Why did this change?

Reply:
For some reason, +15552143356 is considered an invalid number. I believe we filed a bug for that. 
Either way, +14255550123 looks more like a real number according to the server, so it's giving us the 404 not found instead.